doc/deployment/deploy-to-azure-web-app-tomcat-using-jenkins.md

```diff
@@ -0,0 +1,64 @@
+# Deploy to Azure Web App for Tomcat using Jenkins
+
+This tutorial shows you how to deploy a Java app to Azure Web App for Tomcat using [Azure App Service Plugin](https://wiki.jenkins.io/display/JENKINS/Azure+App+Service+Plugin).
+Below are the major steps in this tutorial.
+- [Create Azure Web App](#create-app)
+- [Prepare Jenkins server](#prepare)
+- [Create job](#create-job)
+- [Build and Deploy Java Application to Azure Web App](#deploy)
+- [Clean Up Resources](#clean-up)
+
+## <a name="create-app"></a>Create Azure Web App
```

Comment:
In this session, it should cover how to create a "Linux" based App Service and chosse "Tomcat" as runtime. It now points to a Windows based App Service.

Reply:
Suggest we also tell the audience which tomcat versions are supported now. (e.g. Tomcat 8.5 / 9.0 in Preview)

Reply:
fixed.